#! /bin/sh /usr/share/dpatch/dpatch-run
## reactivate_recommended_driver.dpatch by  <till.kamppeter@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: CUPS removes the "(recommended)" comments of the NickNames of Foomatic
## DP: PPDs when listing available PPDs. This patch removes this remocval
## DP: action.

@DPATCH@
diff -urNad cups-1.4.0~svn8773~/scheduler/cups-driverd.cxx cups-1.4.0~svn8773/scheduler/cups-driverd.cxx
--- cups-1.4.0~svn8773~/scheduler/cups-driverd.cxx	2009-08-23 12:16:58.000000000 +0200
+++ cups-1.4.0~svn8773/scheduler/cups-driverd.cxx	2009-08-23 18:33:34.000000000 +0200
@@ -211,7 +211,6 @@
 	const char *scheme)		/* I - PPD scheme */
 {
   ppd_info_t	*ppd;			/* PPD */
-  char		*recommended;		/* Foomatic driver string */
 
 
  /*
@@ -250,15 +249,6 @@
   strlcpy(ppd->record.scheme, scheme, sizeof(ppd->record.scheme));
 
  /*
-  * Strip confusing (and often wrong) "recommended" suffix added by
-  * Foomatic drivers...
-  */
-
-  if ((recommended = strstr(ppd->record.make_and_model,
-                            " (recommended)")) != NULL)
-    *recommended = '\0';
-
- /*
   * Add the PPD to the PPD arrays...
   */
 
